docs/kb/connect-blockchain.md

```diff
@@ -0,0 +1,266 @@
+---
+description: Connect to Blockchain with RPC
+order: 10
+---
+
+# Connect to Blockchain with RPC
+
+**Learn how to connect your Blockchain to an independent application with RPC**
+
+## Step 1
+- **Scaffold a new Blockchain using `starport`**
+```zsh
+starport scaffold chain github.com/alijnmerchant21/blog
+```
+
+- **Scaffold a new list post with title and body**
+```zsh
+starport s list post title body
+```
+
+**Awesome! You have scaffolded a Blockchain with list. Lets us start the chain.**
+```zsh
+starport chain serve
+```
+
+## Step 2
+**In a new directory (outside the blockchain directory) create two files - `go.main` and `go.mod`**
+```zsh
+touch main.go && touch go.mod
+```
+**`go.main`**
+
+- **Import files**
+
+```go
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/alijnmerchant21/blog/x/blog/types"
+	blogtypes "github.com/alijnmerchant21/blog/x/blog/types"
+	"github.com/tendermint/starport/starport/pkg/cosmosclient"
+)
+```
+***Tip:** Use your own module instead of `github.com/alijnmerchant21/blog/x/blog/types`*
+
+- **Instance of cosmosclient.**
+
+```go
+func main() {
+
+	cosmos, err := cosmosclient.New(context.Background())
+	if err != nil {
+		log.Fatal(err)	
+	}
+```
+- **Using the account name Alice (obtained while scaffolding the blockchain), define the account address**
+
+```go
+accountName := "alice"
+address, err := cosmos.Address(accountName)
+	if err != nil {
+		log.Fatal(err)
+	}
+```
+
+- **`Create` a new post, `broadcast` it on the chain and `display` the results**
+
+```go
+// Create a post
+msg := &types.MsgCreatePost {
+	Creator: address.String(),
+	Title: "Hello Ali",
+	Body: "This is the first post",
+}
+
+// Broadcast the post
+txResp, err := cosmos.BroadcastTx(accountName, msg)
+if err != nil {
+	log.Fatal(err)
+}
+fmt.Print("MsgCreatePost:\n\n")
+fmt.Println(txResp)
+
+// Display the results
+queryClient := blogtypes.NewQueryClient(cosmos.Context)
+queryResp, err := queryClient.Post(context.Background(), &types.QueryGetPostRequest{})
+
+	if err != nil {
+		log.Fatal(err)
+	}
+	fmt.Print("\n\nUser:\n\n")
+	fmt.Println(queryResp)
+
+// Tip: To display result of all users, replace POST with POSTALL
+queryResp, err := queryClient.PostAll(context.Background(), &types.QueryAllPostRequest{})
+```
+
+- **Add `go.mod` file**
+```go
+module github.com/tendermint/starport/local_test/client
+
+go 1.17
+
+require (
+	github.com/alijnmerchant21/blog v0.0.0-20210825193134-b6859adfa282
```

Comment:
when you do `go get github.com/alijnmerchant21/blog` it should give you the version.

Some like this: 

<img width="854" alt="Screen Shot 2021-09-10 at 4 36 31 PM" src="https://user-images.githubusercontent.com/44069404/132914773-08291b9b-30a4-44ce-bf90-bc60da1d7d93.png">

Reply:
Also, since we are using `cosmonaut` instead of `alijnmerchant21` the version no longer makes sense. Also, this is a `go` thing and not starport thing. So we shouldn't be too concerned about it..